.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
Need to put a .gitignore. Take a look here: https://github.com/PauloFer1/pf-onboard/blob/master/.gitignore

Reply:
I created one after opening this PR and noticing the need, but mine is a lot simpler. I will update it with yours.